packages/ngtools/webpack/src/ivy/plugin.ts

```diff
@@ -536,9 +536,7 @@ export class AngularWebpackPlugin {
 
       // NOTE: Workaround to fix stale reuse program. Can be removed once fixed upstream.
       // tslint:disable-next-line: no-any
-      (angularProgram as any).reuseTsProgram =
-        // tslint:disable-next-line: no-any
-        angularCompiler.getNextProgram?.() || (angularCompiler as any).getCurrentProgram?.();
+      (angularProgram as any).reuseTsProgram = angularCompiler.getCurrentProgram();
```

Comment:
This workaround can be removed completely now.
Upstream fix: https://github.com/angular/angular/pull/41289

Reply:
Updated.